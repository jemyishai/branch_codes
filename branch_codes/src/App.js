import React from 'react';
import logo from './logo.svg';
import './App.css';
import branchCodes from './branch_codes.js'
import Codes from './components/codes.js'

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
        <Codes codes={branchCodes} />
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
