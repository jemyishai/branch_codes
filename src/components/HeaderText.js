import React from "react";

const HeaderText = () => {
  return (
    <div>
      <p className="directions-padding">
        NYPL Branch Code <br />Directory
      </p>
    </div>
  );
};

export default HeaderText;
