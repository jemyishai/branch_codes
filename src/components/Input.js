import React from "react";

let Input = (props) => (
  <div className="nypl-text-field">
    <input
      id="keyword-text"
      type="text"
      focus="true"
      placeholder={props.placeholder}
      className=""
      pattern=""
      onChange={props.change}
    />
  </div>
);

export default Input;
