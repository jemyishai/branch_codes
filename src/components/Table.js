import React from "react";

let Table = ({branchInfo:branchCodes, filter:filtering}) => {
  return (
    <div>
      <table className="nypl-basic-table">
        <thead>
          <tr>
            <th scope="col">Branch</th>
            <th scope="col">Code</th>
          </tr>
        </thead>
        <tbody>
          {branchCodes.filter(filtering).map((object) => (
            <tr key={object.branch +1}>
              <td >{object.branch}</td>
              <td >{object.code}</td>
            </tr>
          ))}
        </tbody>
      </table>
    </div>
  );
};

export default Table;
