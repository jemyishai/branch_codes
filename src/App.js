import React from 'react';
import './App.css';
import data from './codesData/formattedData.js';
import Codes from './components/Codes.js';
import Logo from './components/Logo';
import HeaderText from './components/HeaderText';

function App() {
	return (
		<div
		className="inner-padding">
			<div className="nypl-homepage-hero" >
				<div className="box">
					<Logo />
					<HeaderText />
				</div>
			<Codes branches={data}  />
			</div>
		</div>
	);
}

{/* need to add a warning around regexp entries */}

export default App;
