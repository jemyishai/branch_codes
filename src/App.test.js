import React from "react";
import ReactDOM from "react-dom";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import App from "./App";
import Input from "./components/Input";
import HeaderText from "./components/HeaderText";
import Table from "./components/Table";
import { mount } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import TestUtils from "react-dom/test-utils";

configure({ adapter: new Adapter() });

test("basic title check", () => {
  const { getByText } = render(<HeaderText />);
  const linkElement = getByText(/Branch Code/gi);
  expect(linkElement).toBeInTheDocument();
});

it("should take a snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment(<App />)).toMatchSnapshot();
});

it("should recieve right props", function () {
  const component = mount(<Input placeholder="directions" />);
  expect(component.prop("placeholder")).toEqual("directions");
});

//simulate onChange to Input
//quite unfinished

// it("triggers the onChange function by entering text", () => {
//   const wrapper = document.createElement("div");
//   const onChange = jest.fn((data) => data);
//   ReactDOM.render(
//     <Input onChange={onChange}>
//       <Table Branch="Branch" Code="Code" />;
//     </Input>,
//     wrapper
//   );
//   const input = wrapper.querySelector("input");
//   const form = wrapper.querySelector("form");
//   TestUtils.Simulate.change(input, { target: { value: "67th" } });
//   TestUtils.Simulate.submit(form);
//   expect(onChange).toHaveBeenCalledTimes(1);
//   expect(onChange.mock.results[0].value).toEqual({ Branch: "67th" });
// });


//test the regex with random inputs
//test the regex filter with special characters

//test state change
