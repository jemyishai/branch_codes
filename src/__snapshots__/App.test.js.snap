// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="inner-padding"
  >
    <div
      class="nypl-homepage-hero"
    >
      <div
        class="box"
      >
        <div>
          <svg
            aria-hidden="true"
            class="header-mobile-logo svgIcon"
            data-reactid="14"
            focusable="false"
            height="60"
            viewBox="0 0 30 30"
            width="60"
          >
            <title
              data-reactid="15"
            >
              NYPL Lion Logo
            </title>
            <ellipse
              cx="12.503"
              cy="12.485"
              data-reactid="16"
              fill="#FFF"
              rx="12.251"
              ry="12.24"
            />
            <path
              d="M13.048,8.85a4.934,4.934,0,0,1,.879.322,0.893,0.893,0,0,0,.475.263,0.771,0.771,0,0,0,.352-0.609,1.481,1.481,0,0,0-.076-0.837,1.18,1.18,0,0,0-1.119-.351,2.931,2.931,0,0,0-.773.123c-0.27.082-.644,0.263-0.486,0.638A1.2,1.2,0,0,0,13.048,8.85Z"
              data-reactid="17"
            />
            <path
              d="M12.444,0A12.5,12.5,0,1,0,25,12.5,12.468,12.468,0,0,0,12.444,0ZM5.15,21.271a1.841,1.841,0,0,1-.457-0.562c-1.06-1.7-1.658-7.7-.287-9.746,0.434-.714.9-0.386,0.744,0.17a4.505,4.505,0,0,0,.5,3.278c0.949,2,3.873,4.771,4.646,5.777a7.852,7.852,0,0,1,1.764,3.319c-0.006.258-.059,0.427-0.516,0.386A11.339,11.339,0,0,1,5.15,21.271Zm18.344-5.7c-0.094.293-.205,0.661-0.445,0.492a10.744,10.744,0,0,0-2.39-1.317c-0.053-.012-0.047-0.082-0.029-0.123a1.67,1.67,0,0,0,.129-0.468,1.228,1.228,0,0,1,.228-0.41,4.186,4.186,0,0,0,.434-1.5,3.646,3.646,0,0,0-.07-1.188A2.7,2.7,0,0,1,21.2,10.53c0-.17.082-0.345,0.1-0.544a1.614,1.614,0,0,0-1.072-1.235c-0.9-.416-1.851-0.79-2.818-1.305a11.027,11.027,0,0,1-1.424-1.258,10.435,10.435,0,0,0-2.437-1.054,0.228,0.228,0,0,1-.193-0.193,5.677,5.677,0,0,0-2.127-3.3c-0.4-.31.047-0.486,0.6-0.515A11.389,11.389,0,0,1,23.494,15.57Zm-3.527-3.834c-0.006-.047-0.023-0.193-0.023-0.222a0.6,0.6,0,0,1,.24-0.246,2.091,2.091,0,0,1,.334-0.234c0.029-.018.053,0.023,0.059,0.035a3.181,3.181,0,0,1-.029,2.254c-0.029.059-.076,0.082-0.094,0.041a1.454,1.454,0,0,0-.492-0.615,0.115,0.115,0,0,1-.035-0.1A2.749,2.749,0,0,0,19.967,11.736ZM9.491,6.4a3.811,3.811,0,0,1,3.029-.433,13.8,13.8,0,0,1,2.15.784c0.685,0.316,1.172.9,1.81,1.247,0.8,0.445,1.91.656,2.76,1.071a0.8,0.8,0,0,1,.5.451,3.059,3.059,0,0,1-1.623-.023,0.524,0.524,0,0,0-.615.094,0.906,0.906,0,0,0,.059.749,0.979,0.979,0,0,0,.469.509c0.275,0.129.656,0.135,0.908,0.281a1.227,1.227,0,0,1,.182,1.6,2.206,2.206,0,0,1-1.746.4,5.289,5.289,0,0,0-2,.105,2.328,2.328,0,0,0-1.043,1,0.12,0.12,0,0,1-.17.023c-1.775-1.065-4.019-1.616-5.214-3.307a3.638,3.638,0,0,1-.58-1.528A3.018,3.018,0,0,1,9.491,6.4ZM6.72,3.214c-0.352-.041-0.357-0.3-0.205-0.4a8.284,8.284,0,0,1,1.623-.837A0.8,0.8,0,0,1,8.589,1.9a4.956,4.956,0,0,1,2.086.972c1.043,0.743,1.974,2.16,1.353,2.043a5.866,5.866,0,0,0-.68-0.1c-0.469-.041-0.779.006-1-0.018a0.434,0.434,0,0,1-.234-0.123A5.867,5.867,0,0,0,6.72,3.214Zm9.292,11.473a0.675,0.675,0,0,1,.3-0.41,3.043,3.043,0,0,1,1.242-.222,3.994,3.994,0,0,0,1.26-.2,0.773,0.773,0,0,1,.691-0.217,0.5,0.5,0,0,1,.264.322,1.25,1.25,0,0,1,.07.486,13.41,13.41,0,0,1-.58,1.352,0.451,0.451,0,0,1-.07.246,2.132,2.132,0,0,1-1.652.217,2.074,2.074,0,0,1-.592-0.1,1.145,1.145,0,0,1-.293-0.24,6.619,6.619,0,0,1-.51-0.544,0.851,0.851,0,0,1-.228-0.293A1.2,1.2,0,0,1,16.012,14.686ZM4.09,4.812a0.521,0.521,0,0,1,.27-0.17,6.908,6.908,0,0,1,4.365.369C8.982,5.128,9.1,5.286,8.929,5.4a8.935,8.935,0,0,0-1.236.89,0.562,0.562,0,0,1-.4.082,6.571,6.571,0,0,0-4.1.486C2.883,6.983,2.6,6.808,2.742,6.562A10.008,10.008,0,0,1,4.09,4.812Zm-2.818,5.45a0.49,0.49,0,0,1,.123-0.3,7.869,7.869,0,0,1,4.412-2.54,0.628,0.628,0,0,1,.644.111c0.1,0.24-.1.38-0.34,0.515-4.166,2.488-3.873,6.187-3.914,7.7,0.012,0.62-.434.732-0.545,0.439A10.877,10.877,0,0,1,1.271,10.261Zm5.25,2.909a4.944,4.944,0,0,1,.07-4c0.164-.31.322-0.509,0.533-0.451,0.228,0.064.281,0.293,0.311,0.726,0.228,3.565,2.39,4.771,5.1,6.029a15.622,15.622,0,0,1,6.615,5.368c0.311,0.439.352,0.7,0.006,0.954a11.145,11.145,0,0,1-4.019,1.826c-0.246.059-.5,0.012-0.727-0.55C12.122,17.168,8.279,17.437,6.521,13.17Zm14.19,7.252c-0.352.345-.545,0.076-0.662-0.146a10.28,10.28,0,0,0-1.734-2.745,0.178,0.178,0,0,1,.164-0.3,1.287,1.287,0,0,0,.691-0.111,1.383,1.383,0,0,0,.633-0.9c0.1-.339.1-0.445,0.311-0.462a0.632,0.632,0,0,1,.205.023,2.5,2.5,0,0,1,.732.433,6.868,6.868,0,0,1,1.412,1.539,0.4,0.4,0,0,1-.047.4A11.284,11.284,0,0,1,20.711,20.423Z"
              data-reactid="18"
            />
          </svg>
        </div>
        <div>
          <p
            class="directions-padding"
          >
            NYPL Branch Code 
            <br />
            Directory
          </p>
        </div>
      </div>
      <div>
        <div
          class="centered"
        >
          <div
            class="nypl-text-field"
          >
            <input
              class=""
              focus="true"
              id="keyword-text"
              pattern=""
              placeholder="Seach branch name ie... 'Aguilar'"
              type="text"
            />
          </div>
        </div>
        <div>
          <table
            class="nypl-basic-table"
          >
            <thead>
              <tr>
                <th
                  scope="col"
                >
                  Branch
                </th>
                <th
                  scope="col"
                >
                  Code
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  115th Street
                </td>
                <td>
                  HU
                </td>
              </tr>
              <tr>
                <td>
                  125th Street
                </td>
                <td>
                  HD
                </td>
              </tr>
              <tr>
                <td>
                  39th ST - HR - Finance - Marketing
                </td>
                <td>
                  TN
                </td>
              </tr>
              <tr>
                <td>
                  53rd Street
                </td>
                <td>
                  FT
                </td>
              </tr>
              <tr>
                <td>
                  58th Street
                </td>
                <td>
                  FE
                </td>
              </tr>
              <tr>
                <td>
                  67th Street
                </td>
                <td>
                  SS
                </td>
              </tr>
              <tr>
                <td>
                  96th Street
                </td>
                <td>
                  NS
                </td>
              </tr>
              <tr>
                <td>
                  Aguilar
                </td>
                <td>
                  AG
                </td>
              </tr>
              <tr>
                <td>
                  Allerton
                </td>
                <td>
                  AL
                </td>
              </tr>
              <tr>
                <td>
                  Andrew Heiskell
                </td>
                <td>
                  LB
                </td>
              </tr>
              <tr>
                <td>
                  Battery Park
                </td>
                <td>
                  BT
                </td>
              </tr>
              <tr>
                <td>
                  Baychester
                </td>
                <td>
                  BA
                </td>
              </tr>
              <tr>
                <td>
                  Belmont
                </td>
                <td>
                  BE
                </td>
              </tr>
              <tr>
                <td>
                  Bloomingdale
                </td>
                <td>
                  BL
                </td>
              </tr>
              <tr>
                <td>
                  Bookmobile 1
                </td>
                <td>
                  B1
                </td>
              </tr>
              <tr>
                <td>
                  Bookmobile 2
                </td>
                <td>
                  B2
                </td>
              </tr>
              <tr>
                <td>
                  Bookmobile 3
                </td>
                <td>
                  B3
                </td>
              </tr>
              <tr>
                <td>
                  Bronx Library Center
                </td>
                <td>
                  BC
                </td>
              </tr>
              <tr>
                <td>
                  Castle Hill
                </td>
                <td>
                  CT
                </td>
              </tr>
              <tr>
                <td>
                  Chatham Square
                </td>
                <td>
                  CH
                </td>
              </tr>
              <tr>
                <td>
                  City Island
                </td>
                <td>
                  CI
                </td>
              </tr>
              <tr>
                <td>
                  Clason's Point
                </td>
                <td>
                  CP
                </td>
              </tr>
              <tr>
                <td>
                  Collection Rebalancing
                </td>
                <td>
                  XR
                </td>
              </tr>
              <tr>
                <td>
                  Columbus
                </td>
                <td>
                  CS
                </td>
              </tr>
              <tr>
                <td>
                  Countee Cullen
                </td>
                <td>
                  HT
                </td>
              </tr>
              <tr>
                <td>
                  Dongan Hills
                </td>
                <td>
                  DH
                </td>
              </tr>
              <tr>
                <td>
                  Donnell Library Center
                </td>
                <td>
                  DLC
                </td>
              </tr>
              <tr>
                <td>
                  E-Branch
                </td>
                <td>
                  EB
                </td>
              </tr>
              <tr>
                <td>
                  Eastchester
                </td>
                <td>
                  EA
                </td>
              </tr>
              <tr>
                <td>
                  Edenwald
                </td>
                <td>
                  EW
                </td>
              </tr>
              <tr>
                <td>
                  Electronic Material for Children
                </td>
                <td>
                  IJ
                </td>
              </tr>
              <tr>
                <td>
                  Electronic Materials for Adults
                </td>
                <td>
                  IA
                </td>
              </tr>
              <tr>
                <td>
                  Epiphany
                </td>
                <td>
                  EP
                </td>
              </tr>
              <tr>
                <td>
                  Fifty-Third
                </td>
                <td>
                  FT
                </td>
              </tr>
              <tr>
                <td>
                  Fort Washington
                </td>
                <td>
                  FW
                </td>
              </tr>
              <tr>
                <td>
                  Francis Martin
                </td>
                <td>
                  FX
                </td>
              </tr>
              <tr>
                <td>
                  George Bruce
                </td>
                <td>
                  BR
                </td>
              </tr>
              <tr>
                <td>
                  Grand Central
                </td>
                <td>
                  GC
                </td>
              </tr>
              <tr>
                <td>
                  Grand Concourse
                </td>
                <td>
                  GD
                </td>
              </tr>
              <tr>
                <td>
                  Great Kills
                </td>
                <td>
                  GK
                </td>
              </tr>
              <tr>
                <td>
                  Hamilton Fish Park
                </td>
                <td>
                  HF
                </td>
              </tr>
              <tr>
                <td>
                  Hamilton Grange
                </td>
                <td>
                  HG
                </td>
              </tr>
              <tr>
                <td>
                  Harlem
                </td>
                <td>
                  HL
                </td>
              </tr>
              <tr>
                <td>
                  High Bridge
                </td>
                <td>
                  HB
                </td>
              </tr>
              <tr>
                <td>
                  Hudson Park
                </td>
                <td>
                  HP
                </td>
              </tr>
              <tr>
                <td>
                  Huguenot Park
                </td>
                <td>
                  HK
                </td>
              </tr>
              <tr>
                <td>
                  Humanities and Social Sciences
                </td>
                <td>
                  CRL
                </td>
              </tr>
              <tr>
                <td>
                  Hunt's Point
                </td>
                <td>
                  HS
                </td>
              </tr>
              <tr>
                <td>
                  Inter-Library Loan
                </td>
                <td>
                  XFILL
                </td>
              </tr>
              <tr>
                <td>
                  Inwood
                </td>
                <td>
                  IN
                </td>
              </tr>
              <tr>
                <td>
                  Jefferson Market
                </td>
                <td>
                  JM
                </td>
              </tr>
              <tr>
                <td>
                  Jerome Park
                </td>
                <td>
                  JP
                </td>
              </tr>
              <tr>
                <td>
                  Kingsbridge
                </td>
                <td>
                  KB
                </td>
              </tr>
              <tr>
                <td>
                  Kips Bay
                </td>
                <td>
                  KP
                </td>
              </tr>
              <tr>
                <td>
                  LSC Educator Collection
                </td>
                <td>
                  ED
                </td>
              </tr>
              <tr>
                <td>
                  Library Services Center
                </td>
                <td>
                  LS
                </td>
              </tr>
              <tr>
                <td>
                  Macomb's Bridge
                </td>
                <td>
                  MB
                </td>
              </tr>
              <tr>
                <td>
                  Mariner's Harbor
                </td>
                <td>
                  MN
                </td>
              </tr>
              <tr>
                <td>
                  Melrose
                </td>
                <td>
                  ME
                </td>
              </tr>
              <tr>
                <td>
                  Mid-Manhattan
                </td>
                <td>
                  MM
                </td>
              </tr>
              <tr>
                <td>
                  Morningside Heights
                </td>
                <td>
                  CL
                </td>
              </tr>
              <tr>
                <td>
                  Morris Park
                </td>
                <td>
                  MP
                </td>
              </tr>
              <tr>
                <td>
                  Morrisania
                </td>
                <td>
                  MR
                </td>
              </tr>
              <tr>
                <td>
                  Mosholu
                </td>
                <td>
                  MO
                </td>
              </tr>
              <tr>
                <td>
                  Mott Haven
                </td>
                <td>
                  MH
                </td>
              </tr>
              <tr>
                <td>
                  Muhlenberg
                </td>
                <td>
                  MU
                </td>
              </tr>
              <tr>
                <td>
                  Mulberry Street
                </td>
                <td>
                  ML
                </td>
              </tr>
              <tr>
                <td>
                  NYCHA
                </td>
                <td>
                  NYJ
                </td>
              </tr>
              <tr>
                <td>
                  NYCHA Collection
                </td>
                <td>
                  NYY
                </td>
              </tr>
              <tr>
                <td>
                  NYPL Children's Center @ 42nd
                </td>
                <td>
                  MAJ
                </td>
              </tr>
              <tr>
                <td>
                  NYPL Research Libraries
                </td>
                <td>
                  RC
                </td>
              </tr>
              <tr>
                <td>
                  New Amsterdam
                </td>
                <td>
                  LM
                </td>
              </tr>
              <tr>
                <td>
                  New Dorp
                </td>
                <td>
                  ND
                </td>
              </tr>
              <tr>
                <td>
                  Ninety-sixth
                </td>
                <td>
                  NS
                </td>
              </tr>
              <tr>
                <td>
                  OFF SITE - ReCAP Partner
                </td>
                <td>
                  OS
                </td>
              </tr>
              <tr>
                <td>
                  One Hundred Twenty-Fifth Street
                </td>
                <td>
                  HD
                </td>
              </tr>
              <tr>
                <td>
                  One Hundred and Fifteenth Street
                </td>
                <td>
                  HU
                </td>
              </tr>
              <tr>
                <td>
                  Ottendorfer
                </td>
                <td>
                  OT
                </td>
              </tr>
              <tr>
                <td>
                  Parkchester
                </td>
                <td>
                  PK
                </td>
              </tr>
              <tr>
                <td>
                  Pelham Bay
                </td>
                <td>
                  PM
                </td>
              </tr>
              <tr>
                <td>
                  Pelham Parkway-Van Nest
                </td>
                <td>
                  VN
                </td>
              </tr>
              <tr>
                <td>
                  Performing Arts
                </td>
                <td>
                  MY
                </td>
              </tr>
              <tr>
                <td>
                  Performing Arts Library at Lincoln Center
                </td>
                <td>
                  MY
                </td>
              </tr>
              <tr>
                <td>
                  Port Richmond
                </td>
                <td>
                  PR
                </td>
              </tr>
              <tr>
                <td>
                  Research Electronic Materials for Adults
                </td>
                <td>
                  IARCH
                </td>
              </tr>
              <tr>
                <td>
                  Research Libraries - On Order
                </td>
                <td>
                  XXX
                </td>
              </tr>
              <tr>
                <td>
                  Richmondtown
                </td>
                <td>
                  RT
                </td>
              </tr>
              <tr>
                <td>
                  Riverdale
                </td>
                <td>
                  RD
                </td>
              </tr>
              <tr>
                <td>
                  Riverside
                </td>
                <td>
                  RS
                </td>
              </tr>
              <tr>
                <td>
                  Roosevelt Island
                </td>
                <td>
                  RI
                </td>
              </tr>
              <tr>
                <td>
                  SASB
                </td>
                <td>
                  MA
                </td>
              </tr>
              <tr>
                <td>
                  SIBL - Science Industry and Business
                </td>
                <td>
                  SL
                </td>
              </tr>
              <tr>
                <td>
                  SIBL Research Electronic Material for Adults
                </td>
                <td>
                  IASLR
                </td>
              </tr>
              <tr>
                <td>
                  Schomburg Center
                </td>
                <td>
                  SC
                </td>
              </tr>
              <tr>
                <td>
                  Science Industry and Business
                </td>
                <td>
                  SL
                </td>
              </tr>
              <tr>
                <td>
                  Sedgwick
                </td>
                <td>
                  SD
                </td>
              </tr>
              <tr>
                <td>
                  Seward Park
                </td>
                <td>
                  SE
                </td>
              </tr>
              <tr>
                <td>
                  Sixty Seventh
                </td>
                <td>
                  SS
                </td>
              </tr>
              <tr>
                <td>
                  Soundview
                </td>
                <td>
                  SV
                </td>
              </tr>
              <tr>
                <td>
                  South Beach
                </td>
                <td>
                  SB
                </td>
              </tr>
              <tr>
                <td>
                  Spuyten Duyvil
                </td>
                <td>
                  DY
                </td>
              </tr>
              <tr>
                <td>
                  St. Agnes
                </td>
                <td>
                  SA
                </td>
              </tr>
              <tr>
                <td>
                  St. George
                </td>
                <td>
                  SG
                </td>
              </tr>
              <tr>
                <td>
                  Stapleton
                </td>
                <td>
                  ST
                </td>
              </tr>
              <tr>
                <td>
                  Stavros Niarchos Foundation Library
                </td>
                <td>
                  SN
                </td>
              </tr>
              <tr>
                <td>
                  Stephen A. Schwarzmann Building
                </td>
                <td>
                  MA
                </td>
              </tr>
              <tr>
                <td>
                  Terence Cardinal Cooke-Cathedral
                </td>
                <td>
                  CA
                </td>
              </tr>
              <tr>
                <td>
                  Terrence Cardinal Cooke-Cathedral
                </td>
                <td>
                  CA
                </td>
              </tr>
              <tr>
                <td>
                  Thirty ninth
                </td>
                <td>
                  TN
                </td>
              </tr>
              <tr>
                <td>
                  Throg's Neck
                </td>
                <td>
                  TG
                </td>
              </tr>
              <tr>
                <td>
                  Todt Hill-Westerleigh
                </td>
                <td>
                  TH
                </td>
              </tr>
              <tr>
                <td>
                  Tompkins Square
                </td>
                <td>
                  TS
                </td>
              </tr>
              <tr>
                <td>
                  Tottenville
                </td>
                <td>
                  TV
                </td>
              </tr>
              <tr>
                <td>
                  Tremont
                </td>
                <td>
                  TM
                </td>
              </tr>
              <tr>
                <td>
                  Van Cortlandt
                </td>
                <td>
                  VC
                </td>
              </tr>
              <tr>
                <td>
                  Van Nest
                </td>
                <td>
                  VN
                </td>
              </tr>
              <tr>
                <td>
                  Wakefield
                </td>
                <td>
                  WK
                </td>
              </tr>
              <tr>
                <td>
                  Washington Heights
                </td>
                <td>
                  WH
                </td>
              </tr>
              <tr>
                <td>
                  Webster
                </td>
                <td>
                  WB
                </td>
              </tr>
              <tr>
                <td>
                  West Farms
                </td>
                <td>
                  WF
                </td>
              </tr>
              <tr>
                <td>
                  West New Brighton
                </td>
                <td>
                  NB
                </td>
              </tr>
              <tr>
                <td>
                  Westchester Square
                </td>
                <td>
                  WT
                </td>
              </tr>
              <tr>
                <td>
                  Woodlawn Heights
                </td>
                <td>
                  WL
                </td>
              </tr>
              <tr>
                <td>
                  Woodstock
                </td>
                <td>
                  WO
                </td>
              </tr>
              <tr>
                <td>
                  World Land Orders
                </td>
                <td>
                  CCD
                </td>
              </tr>
              <tr>
                <td>
                  Yorkville
                </td>
                <td>
                  YV
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
